File: output/qwen_qwen3_coder_next/zero/python/Prêmio do Milhão.py
n = int(input())
acessos = []
for _ in range(n):
    acessos.append(int(input()))

total = 0
dias = 0
for i in range(n):
    total += acessos[i]
    dias += 1
    if total >= 1000000:
        break

print(dias)

File: output/csbc_semish_2026/mistralai_mistral_small_2603/zero/cpp/Prêmio do Milhão.cpp

#include <iostream>
using namespace std;

int main() {
    int N;
    cin >> N;
    long long total = 0;
    int dias = 0;
    for (int i = 0; i < N; i++) {
        int A;
        cin >> A;
        total += A;
        dias++;
        if (total >= 1000000) {
            break;
        }
    }
    cout << dias << endl;
    return 0;
}
